POLICIES.md

```diff
@@ -107,13 +110,37 @@ spec:
           trustServer: "https://icr.io:4443" # Optional, custom trust server for repository
 ```
 
-For more information, see the [IBM Cloud docs](https://cloud.ibm.com/docs/Registry?topic=Registry-security_enforce#customize_policies).
+For more information, see [Customizing policies](#customizing-policies).
 
+### Specifying trusted content signers in custom policies
```

Comment:
Let's drop this one level of heading so that it fits under the `trust` heading above.

```suggestion
#### Specifying trusted content signers in custom policies
```

Reply:
done